examples/recovery/transliterated.json

```diff
@@ -0,0 +1,21 @@
+{
+  "ver": "1.3.0",
+  "nam": {
+    "fn": "ابو بكر محمد بن زكريا الرازي",
+    "fnt": "ABU<BAKR<MOHAMMED<IBN<ZAKARIA<AL<RAZI",
+    "gn": "ناصر",
+    "gnt": "NASR"
+  },
+  "dob": "1964",
```

Comment:
All examples use just the year. My guess is that most countries will use a complete date of birth. So the examples should reflect that fact. Let's use year-only for one example, but complete for the others.

Reply:
I'm currently expanding the test cases under _test_; I'm open to making the examples into the canonical (golden standard) form with acceptable variants being represented under test. That way keep the number of examples and variance between them very low. With the test cases being broader.

Thoughts @gabywh @martin-lindstrom ?

Reply:
Checking for at least (so: precisely) 1 event is done using business rules; cf.: https://github.com/ehn-dcc-development/dgc-business-rules/blob/main/rulesets/EU/template-ruleset.json#L66

Reply:
> Checking for at least (so: precisely) 1 event is done using business rules; cf.: https://github.com/ehn-dcc-development/dgc-business-rules/blob/main/rulesets/EU/template-ruleset.json#L66

Meaning? That the schema doesn't have to represent that rule?

Reply:
Meaning that the “silly corner case” would be covered by this rule. (I don't think it's that silly, TBH: without checking it explicitly, “free-pass” DCCs could be issued.)

Reply:
If a validator is so badly written that it would accept a "free-pass" DCC, then we can't expect it to validate according to the schema either.

So, this means that we can revert back to the 1.2.1 way of expressing it (with the maxItems=1 extension)?

Reply:
Just to be clear. I am fine with this PR as long as the examples are updated, even though I think that the allOf-fix makes the JSON harder to read for very little gain.

Reply:
> Meaning that the “silly corner case” would be covered by this rule. (I don't think it's that silly, TBH: without checking it explicitly, “free-pass” DCCs could be issued.)

You're right - it's not silly... it's non-existent - https://github.com/ehn-dcc-development/ehn-dcc-schema/pull/100#discussion_r649357673

Reply:
Updated! PR for test cases coming in a sec!